src/components/SettingsStrategiesBlock.vue

```diff
@@ -2,16 +2,26 @@
 import { SpaceStrategy } from '@/helpers/interfaces';
 import { clone } from '@snapshot-labs/snapshot.js/src/utils';
 import schemas from '@snapshot-labs/snapshot.js/src/schemas';
+import spaceSchema from '@snapshot-labs/snapshot.js/src/schemas/space.json';
```

Comment:
This seems like a duplicate import. Instead I would create a var like this `const spaceSchema = schemas.space`